plugins/modules/source_control/github/github_repo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2021, Álvaro Torres Cogollo
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.basic import AnsibleModule
+from github import Github, GithubException
+import sys
+
+DOCUMENTATION = r'''
+---
+module: github_repo
+short_description: Manage your repos on Github
+description:
+- Manages Github repos using PyGithub library
```

Comment:
```suggestion
- Manages Github repos using PyGithub library.
```
All `description`s (except `short_description`) should always have complete sentences.